manifests/base/commit.yaml

```
@@ -80,3 +80,11 @@ varReference:
     kind: ImageStream
     apiGroup: image.openshift.io/v1
     name: odh-rstudio-gpu-notebook-image-commit-n-1
+  - path: spec/tags[]/annotations/opendatahub.io\/notebook-build-commit
+    kind: ImageStream
+    apiGroup: image.openshift.io/v1
+    name: odh-rocm-pytorch-notebook-image-commit-n
+  - path: spec/tags[]/annotations/opendatahub.io\/notebook-build-commit
+    kind: ImageStream
+    apiGroup: image.openshift.io/v1
+    name: odh-rocm-pytorch-notebook-image-commit-n
```

```suggestion
    name: odh-rocm-tensorflow-notebook-image-commit-n
```
one of this should be tensorflow.